projects/ngrx.io/content/examples/testing-store/src/app/reducers/auth.reducer.ts

```diff
@@ -0,0 +1,20 @@
+import { createReducer, on } from '@ngrx/store';
+import { AuthActions } from '../actions';
+
+export interface State {
+  loggedIn: boolean;
+}
+
+export const initialState: State = {
+  loggedIn: false,
+};
+
+export const reducer = createReducer<State>(
```

Comment:
The `ons` don't need to be wrapped in an array:

```ts
export const reducer = createReducer<State>(
  initialState,
  on(AuthActions.login, (): State => ({ loggedIn: true })),
  on(AuthActions.logout, (): State => ({ loggedIn: false }))
);
```

This should also resolve the stackblitz error that you're encountering

Reply:
To resolve the stackblitz error, add `jasmine` in https://github.com/ngrx/platform/blob/master/projects/ngrx.io/tools/examples/shared/boilerplate/cli/package.json

Reply:
While you're in that file, you should also add `tslib` - this is the current screen we get with all the examples on next.ngrx.io. Or would this be stackblitz related?


![image](https://user-images.githubusercontent.com/28659384/56852635-dfba3000-691d-11e9-9835-91766cb521eb.png)

Reply:
While you're in that file we should also add `tslib`:

![image](https://user-images.githubusercontent.com/28659384/56852694-b64dd400-691e-11e9-9cc4-c26c87085749.png)

(this change is needed for all the examples - this dep was needed for store migrations)